ts/sagas/startup.ts

```diff
@@ -241,7 +241,7 @@ export function* initializeApplicationSaga(
   // Since the backend.json is done in parallel with the startup saga,
   // we need to synchronize the two tasks, to be sure to have loaded the remote FF
   // before using them.
-  const backendStatus = yield* select(backendStatusSelector);
+  const backendStatus = yield* select(remoteConfigSelector);
```

Comment:
Should we rename `backendStatus` to `remoteConfig` ?

Reply:
applied in [3194010](https://github.com/pagopa/io-app/pull/6296/commits/31940103b5c76793da1c12a38751374fa1b25d92)